lib/internal/Magento/Framework/GraphQl/Query/ErrorHandlerInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Framework\GraphQl\Query;
+
+/**
+ * Interface ErrorHandlerInterface
+ *
+ * @package Magento\Framework\GraphQl\Query
+ */
+interface ErrorHandlerInterface
+{
+    const SERVER_LOG_FILE = 'var/log/graphql/server/exception.log';
```

Comment:
It's should be part of DI configuration, but not hard coded in constant

Reply:
I agree that it seems like something hard-coded, but by the fact the only place where it is used is `di.xml`:
https://github.com/magento/graphql-ce/blob/9027c50f7fa39c6bca79c8023e2191d441a1d937/app/etc/di.xml#L1813

If it is not OK, it`s not a problem for me to get rid of this.

Reply:
@novikor @naydav - I think this ok that we do have these constants in the interface and inject them via di